packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -842,6 +842,8 @@ Adds a SAML 2.0 application. This application is only available to the org that
 
 | Parameter             | Description                                     | DataType                             | Nullable | Unique | Validation                              |
 | --------------------- | ----------------------------------------------- | ------------------------------------ | -------- | -----  | --------------------------------------  |
+| allowMultipleAcsEndpoints       | Determines whether the app allows you to configure multiple ACS URIs                                    | Boolean                                              | FALSE    | FALSE  |                                           |
+| acsEndpoints          | An array of ACS endpoints. You can configure a maximum of 100 endpoints.                                          | Array of [ACS Endpoints](#acs-endpoint-object)       | TRUE     | FALSE  |                                           |
```

Comment:
Remove "of [ACS Endpoints](#acs-endpoint-object) " at the end and do the link like below:

An array of [ACS Endpoints](#acs-endpoint-object). You can configure a maximum of 100 endpoints.